diff --git a/node_modules/react-native-date-picker/.bun-tag-bbde9e6dfcdf534e b/.bun-tag-bbde9e6dfcdf534e
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/ios/DatePicker.m b/ios/DatePicker.m
index e747f3f8d0e3cd8e3f90724e6e9d64e059ba1eed..b5d891c6e70457074ebd9543b8a6cad974596580 100644
--- a/ios/DatePicker.m
+++ b/ios/DatePicker.m
@@ -4,13 +4,13 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
-#import "RCTComponent.h"
+#import <React/RCTComponent.h>
 #import "DatePicker.h"
 
-#import "RCTUtils.h"
-#import "UIView+React.h"
-#import "RCTConvert.h"
-#import "RCTView.h"
+#import <React/RCTUtils.h>
+#import <React/UIView+React.h>
+#import <React/RCTConvert.h>
+#import <React/RCTView.h>
 
 @interface DatePicker ()
 
diff --git a/ios/RNDatePicker.mm b/ios/RNDatePicker.mm
index 7ec2dd20b636e901e747cf8117de2e8ba1b39215..4de23a586c7781fc523cecf32351036324bd75aa 100644
--- a/ios/RNDatePicker.mm
+++ b/ios/RNDatePicker.mm
@@ -1,7 +1,7 @@
 #import "RNDatePicker.h"
 
 #ifdef RCT_NEW_ARCH_ENABLED
-#import "RCTConvert.h"
+#import <React/RCTConvert.h>
 #import <React/RCTConversions.h>
 #import <react/renderer/components/RNDatePickerSpecs/ComponentDescriptors.h>
 #import <react/renderer/components/RNDatePickerSpecs/EventEmitters.h>
@@ -12,9 +12,9 @@
 using namespace facebook::react;
 
 #else
-#import "RCTUtils.h"
-#import "UIView+React.h"
-#import "RCTComponent.h"
+#import <React/RCTUtils.h>
+#import <React/UIView+React.h>
+#import <React/RCTComponent.h>
 #endif
 
 
diff --git a/ios/RNDatePickerManager.mm b/ios/RNDatePickerManager.mm
index da42873cad53e841becb743de6e0d733ab982ce1..f7247b87fbad508f41d6ec915c28ca295935c936 100644
--- a/ios/RNDatePickerManager.mm
+++ b/ios/RNDatePickerManager.mm
@@ -3,7 +3,7 @@
 #import <React/RCTViewManager.h>
 
 #import "RNDatePickerManager.h"
-#import "RCTConvert.h"
+#import <React/RCTConvert.h>
 #import "DatePicker.h"
 #import "RNDatePicker.h"
 
diff --git a/android/build.gradle b/android/build.gradle
index b2803c4f830c53c52831695190c30e5f0a803ddd..ea1069292aa9280cecd9b97fd81f925d6e6c9433 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -51,7 +51,7 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
 }
